index.html

```diff
@@ -169,9 +265,10 @@ <h1>About me</h1>
 					<p> Hello I’m a software developer! I can help you build a product , feature or website Look through
 						some of my work and experience! If you like what you see and have a project you need coded,
 						don’t hestiate to contact me.</p>
-					<div><button type="button" class="aboutMeButton">Get My Resume</button></div>
+					<div><button type="button" class="aboutMeButton footerTopButton">Get My Resume</button></div>
```

Comment:
Great job @dgcuenca 🤝

- [ ] [Optional] Kindly increase the size of this button so that it can replicate the one provided in the Figma design.